src/commands/actor/charge.ts

```diff
@@ -0,0 +1,81 @@
+import { APIFY_ENV_VARS } from '@apify/consts';
+import { Args, Flags } from '@oclif/core';
+
+import { getApifyTokenFromEnvOrAuthFile } from '../../lib/actor.js';
+import { ApifyCommand } from '../../lib/apify_command.js';
+import { getLoggedClient } from '../../lib/utils.js';
+
+/**
+ * This command can be used to charge for a specific event in the pay-per-event Actor run.
+ * - If run locally or with the --test-pay-per-event flag, it will only log the charge request, without actually charging.
+ * - If run in the Actor run on Apify platform without the testing flag, it will charge the specified amount of events.
+ *
+ * Future TODOs:
+ * - Add logic to work with the max charge USD to prevent exceeding the charging limit.
+ * - Add logic to store events in the log dataset for later inspection to aid local development.
+ */
+export class ChargeCommand extends ApifyCommand<typeof ChargeCommand> {
+	static override description = 'Charge for a specific event in the pay-per-event Actor run.\n';
```

Comment:
should this really be ended with a line break @vladfrangu ?

Reply:
This is mostly a design choice, it's not required but if it looks better, go for it. Usually we try to avoid them tho

Reply:
My point is that this has likely no effect, I would expect oclif to do `description.trim()` on the value.

Also I can't find any command right now that would be doing this, so let's be consistent and drop it.